x/emissions/abci.go

```diff
@@ -102,11 +103,13 @@ func DistributeObserverRewards(
 	if len(ballotIdentifiers) == 0 {
 		return nil
 	}
+	ballots := make([]observertypes.Ballot, 0, len(ballotIdentifiers))
```

Comment:
We need the ballot , for the `EmitBallotDeleted` event .

I wanted to concentrate all the deletion logic into `ClearMaturedBallots` which is why I am emitting the event from inside the function instead of from emissons/abci.go directly